### PR TITLE
Add Paid Event Widgets

### PR DESCRIPTION
**Pull request by jlevine18**
Currently ACM Bar Crawl Specific but will make general (especially when Dev's event API is up)

### CODE REVIEW

**Comment by jlevine18** [comment]
Please do not merge until social decides it's ok to start ticket sales

File: src/components/Hero/events.json
Comment (devksingh4) [inline src/components/Hero/events.json:1]: These dates and events are off since there's an old base.

File: src/pages/membership/Event.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        color: 'default'
+      };
+    const isValid = validateNetId(netId);
+    if (isValid && validateNetId(netIdConfirm) && netId === netIdConfirm) {
+      setValidated(true);
+    } else {
+      setValidated(false);
+    }
+    return {
+      text: isValid ? '' : 'Enter a valid NetId',
```
Comment (devksingh4) [inline src/pages/membership/Event.tsx:97]: nitpick: should be NetID

File: src/pages/membership/Event.tsx
Comment (devksingh4) [inline src/pages/membership/Event.tsx:1]: This page has a lot of hardcoded details and still reuses the Paid Member signup modal data. The code should be refactored to be more adaptable.

**Review by devksingh4 — CHANGES_REQUESTED** [review]

File: src/components/Hero/events.json
Comment (jlevine18) [reply src/components/Hero/events.json:1]: Updated. Should be addressed now.

File: src/pages/membership/Event.tsx
Comment (jlevine18) [reply src/pages/membership/Event.tsx:1]: Modularized into JSON. Will replace with API when that comes up.

**Review by devksingh4 — COMMENTED** [review]
LGTM, pending review from Labdhi.

File: src/pages/membership/Event.tsx (hunk 1 above)
Comment (jlevine18) [reply src/pages/membership/Event.tsx]: FYI, this comes from the Member payment page. Updated both.

**Pull request merged**